packages/cashier/src/stores/__tests__/withdraw-store.spec.js

```diff
@@ -91,14 +90,11 @@ describe('WithdrawStore', () => {
     });
 
     it('should set is_withdraw_confirmed', () => {
-        const spyClearVerification = jest.spyOn(withdraw_store.verification, 'clearVerification');
-
         withdraw_store.setIsWithdrawConfirmed(true);
         expect(withdraw_store.is_withdraw_confirmed).toBeTruthy();
         expect(withdraw_store.withdraw_amount).toBe(100);
 
         withdraw_store.setIsWithdrawConfirmed(false);
```

Comment:
Does this action need expect?

Reply:
@hamid-deriv I'm not sure it's the old code, Should I add it? 🤔

Reply:
This line is not testing anything. Maybe it could be removed now.